wolfcrypt/test/test.c

```diff
@@ -18987,7 +18996,11 @@ static wc_test_ret_t rsa_keygen_test(WC_RNG* rng)
     RsaKey genKey[1];
 #endif
     wc_test_ret_t ret;
+#ifndef WOLFSSL_NO_MALLOC
     byte*  der = NULL;
+#else
+    byte der[FOURK_BUF];
```

Comment:
Can this be smaller too?
```suggestion
    byte der[1024];
```